src/components/ReviewConfirm/Summary.vue

```diff
@@ -1,22 +1,36 @@
 <template>
   <div class="summary-container">
-    <SummaryDefineCompany/>
+    <SummaryDefineCompany />
+    <ListPeopleAndRoles :personList="orgPersonList" :isSummary="true" :showErrorSummary="!stepValid"/>
```

Comment:
I realize you are reusing this component from Step 2, which is different from the dedicated SummaryDefineCompany component... I just wanted to say that I think this is OK! (Ie, Define Company needed a new summary component but People and Roles does not.)

Reply:
I also thought on that a bit! Would have loved for everything to be a single component with a flag but i also agree this is 100% situational.